my-app/src/config/const.ts

```diff
@@ -10,7 +10,7 @@ export default options;
 
 type currencyIconTypes = {
   [key: string]: string,
```

Comment:
1. типы принято писать `PascalCase`-ом
2. можно [вот так](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type):
```suggestion
type CurrencyIconTypes = Record<string, string>;
```